src/VisualStudio/Core/Def/Implementation/Library/FindResults/TreeItems/MetadataDefinitionTreeItem.cs

```diff
@@ -40,7 +40,7 @@ public override int GoToSource()
             if (symbol != null && referencingProject != null)
             {
                 var navigationService = _workspace.Services.GetService<ISymbolNavigationService>();
-                return navigationService.TryNavigateToSymbol(symbol, referencingProject, CancellationToken.None)
+                return navigationService.TryNavigateToSymbol(symbol, referencingProject, cancellationToken: CancellationToken.None)
```

Comment:
I see that it weren't passing a cancellation token ever, i also seem to vaguely remember that it had something to do with VS find symbol results window. It'd be nice to document why this is not cancellable here for future reference, if you have the context behind that.. 
Also feel free to ignore, considering the time we've left open in stabilization.